src/plugins/data_view_editor/public/components/rollup_deprecated_warning/rollup_deprecated_warning.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import { EuiCallOut, EuiLink } from '@elastic/eui';
+
+const rollupBetaWarningTitle = i18n.translate(
+  'indexPatternEditor.rollupIndexPattern.deprecationWarning.title',
+  {
+    defaultMessage: 'Deprecated in 8.11.0',
+  }
+);
+
+export const RollupDeprecatedWarning = () => (
+  <EuiCallOut
+    title={rollupBetaWarningTitle}
+    color="warning"
+    iconType="help"
+    data-test-subj="rollupDeprecationCallout"
+  >
+    <FormattedMessage
+      id="indexPatternEditor.rollupDataView.deprecationWarning.textParagraphOne"
+      defaultMessage="Rollups are deprecated and will be removed in a future version. {downsamplingLink} can be used as an alternative."
+      values={{
+        downsamplingLink: (
+          <EuiLink
+            href="https://www.elastic.co/guide/en/elasticsearch/reference/current/downsampling.html"
```

Comment:
Can we use `docLinks` service instead?

Reply:
Good call! Fixed in [33abfcb](https://github.com/elastic/kibana/pull/187505/commits/33abfcb165c60cebff77ea1d938632d6f671066f)